calico_kubernetes/calico_kubernetes.py

```diff
@@ -68,7 +68,7 @@
 # Flag to indicate whether or not to use Calico IPAM.
 # If False, use the default docker container ip address to create container.
 # If True, use libcalico's auto_assign IPAM to create container.
-CALICO_IPAM = os.environ.get('CALICO_IPAM', 'false')
```

Comment:
CALICO_IPAM=TRUE will fail. Should we force `os.environ.get().lower()` here?

(That's an existing bug, so happy to let this through if you don't want to fix as part of this PR)